contracts/VToken.sol

```diff
@@ -1786,6 +1914,28 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         emit NewAccessControlManager(oldAccessControlManager, accessControlManager);
     }
 
+    /**
+     * @notice Sets the utilization threshold for stable rate rebalancing
+     * @param utilizationRateThreshold The utilization rate threshold
+     * @custom:access Only Governance
+     */
+    function setRebalanceUtilizationRateThreshold(uint256 utilizationRateThreshold) external {
+        require(msg.sender == owner(), "only admin can set ACL address");
```

Comment:
I would use the `AccessControlManager` to delegate these changes to VIP's

Reply:
Fixed, can verify at [commit](https://github.com/VenusProtocol/isolated-pools/pull/157/commits/9623c09d414b1d588a9416a98457f4eb8bf3dba5).